README.md

```diff
@@ -104,6 +104,26 @@ parameters:
 
 If the bucket is specified, it will still be created if it does not exist on the backend. Every volume will get its own prefix within the bucket which matches the volume ID. When deleting a volume, also just the prefix will be deleted.
 
+#### Using an existing bucket with custom prefix
+
+If you have an existing bucket and with or without a prefix (subpath), you can specify to use a prefixed configuration by setting the parameters as:
```

Comment:
If you have a `StorageClass` like this, every volume created with it will have the same contents, which is not what you usually expect with k8s volumes. I feel like the docs should explain that a bit more clearly.